lib/java/com/google/android/material/tabs/TabLayout.java

```diff
@@ -719,8 +719,6 @@ public void setSelectedTabIndicatorHeight(int height) {
    * Set the scroll position of the tabs. This is useful for when the tabs are being displayed as
    * part of a scrolling container such as {@link androidx.viewpager.widget.ViewPager}.
    *
-   * <p>Calling this method does not update the selected tab, it is only used for drawing purposes.
```

Comment:
I think we should still keep this part:

"<p>Note that calling this method does not update the selected tab."

Reply:
>"Note that calling this method does not update the selected tab."

But [it does](https://github.com/material-components/material-components-android/blob/b7cfffcdd9d57bc6e9e0564dc0f3a10e84e7efec/lib/java/com/google/android/material/tabs/TabLayout.java#L1908).

Reply:
I guess it means the tab selection won't be changed? Probably rephrase the sentence then?

Reply:
Maybe another time. I'll put this line back then.

Reply:
Hey sorry I didn't take a close look at the code but now I get what you mean. 

The method is actually updating the selected tab and yeah the javadoc is definitely improper. 
Sorry that can you still remove those two lines? And I'll send it for our internal review.

Thanks for your contribution to the codebase, as always. : )

Reply:
Ok, done.

P.S. Well, technically, the comment is not entirely improper, it's just ambiguous. 
I could rephrase it like "Note that calling this method with `updateTabSelection = true` does not actually select a tab at the requested position, but only _draws it as selected_", but I'm not sure this comment is needed here at all.

Reply:
I decided not to remove the comment, but just changed it a little. It still needs some editing, but I believe it's better to leave this task to your reviewers.